odoo/tests/common.py

```diff
@@ -69,6 +69,9 @@
 ADMIN_USER_ID = odoo.SUPERUSER_ID
 
 
+NON_ROLLBACKED_SEQUENCES = {'ir_profile_id_seq', 'ir_sequence_id_seq', 'bus_bus_id_seq'}
```

Comment:
you might want to explain why?

Reply:
This is partially explained in the commit message ;)